### ISSUE TITLE
phpunit test failure: `tests\classes\utilities\ReflectionTest::testTypeReturnsTypeOfReflectedClass`

### ISSUE THREAD

**Issue by sevidmusic** [sevidmusic/DarlingDataManagementSystem#381]
The following test fails inconsistently:

```
tests\classes\utilities\ReflectionTest::testTypeReturnsTypeOfReflectedClass
```

`phpunit` output:

```
PHPUnit 9.5.26 by Sebastian Bergmann and contributors.

Runtime:       PHP 8.1.2
Configuration: /home/darling/dev/php/roady/php.xml
Random Seed:   1668616108

..............................................F................  63 / 107 ( 58%)
............................................                    107 / 107 (100%)

Time: 00:05.105, Memory: 6.00 MB

There was 1 failure:

1) tests\classes\utilities\ReflectionTest::testTypeReturnsTypeOfReflectedClass
The roady\classes\utilities\Reflection implementation's type() method must return the type of the reflected class.
Failed asserting that roady\classes\strings\ClassString Object &00000000000001450000000000000000 (
    'string' => 'roady\classes\strings\UnknownClass'
) matches expected 'roady\interfaces\strings\ClassString'.

/home/darling/dev/php/roady/tests/interfaces/utilities/ReflectionTestTrait.php:281

FAILURES!
Tests: 107, Assertions: 4107, Failures: 1.

```

Look into an fix this asap.

**Comment by sevidmusic** [comment]
Resolved in c796d38b2e0ff0a4607d59bee63711c7575a597a

**Comment by sevidmusic** [comment]
Resolved in c796d38b2e0ff0a4607d59bee63711c7575a597a
